drivers/watchdog/wdt_ti_tps382x.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023, Prevas A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_tps382x
+
+#include <errno.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/watchdog.h>
+#include <zephyr/logging/log.h>
+
+#define LOG_LEVEL CONFIG_WDT_LOG_LEVEL
+
+LOG_MODULE_REGISTER(wdt_ti_tps382x);
+
+struct ti_tps382x_config {
+	struct gpio_dt_spec gpio;
+	int timeout;
+};
+
+static const struct ti_tps382x_config ti_tps382x_config = {
+	.gpio = GPIO_DT_SPEC_INST_GET(0, wdi_gpios),
```

Comment:
You should check the `gpio` member using `gpio_is_ready_dt` function somewhere in the driver initialization.

Reply:
Thanks! Fixed.